var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -172,10 +199,6 @@ def configure_args(self):
 
             if '+proj' in spec:
                 args.append('--with-proj={0}'.format(spec['proj'].prefix))
-                if spec.satisfies('^proj@5.0:5.999'):
-                    args.append('--with-proj5-api=yes')
-                else:
-                    args.append('--with-proj5-api=no')
```

Comment:
Why was this removed?

Reply:
I'm not sure if it's actually necessary, and the flag no longer exists in newer versions.